docs/build/custom.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Custom build
+description: Customize the ONNX Runtime binaries, including building with a reduced set of operators
+parent: Build ONNX Runtime
+nav_order: 7
+redirect_from: /docs/tutorials/mobile/custom-build,/docs/build/reduced,,/docs/tutorials/mobile/limitations
+---
+
+# Build a custom ONNX Runtime package
+{: .no_toc }
+
+The ONNX Runtime package can be customized when the demands of the target environment require it.
+
+The most common scenario for customizing the ONNX Runtime build is for smaller footprint deployments, such as mobile and web.
+
+And the most common mechanism to customize the build is to reduce the set of supported operators in the runtime to only those in the model or models that run in the target environment.
+
+To build a custom ONNX Runtime package, the [build from source](./index.md) instructions apply, with some extra options that are specified below.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Reduce operator set
+
```

Comment:
The 'set' might be confusing here given onnx has the notion of an 'operator set' (generally called the opset). 

'Reduce operator kernels' might be clearer as a kernel is the implementation of the operator for the opset the model requires.

Reply:
Fixed